packages/dev/buildTools/src/copyAssets.ts

```diff
@@ -19,10 +19,11 @@ const processFile = (file: string, options: { isCore?: boolean; basePackageName?
         } else {
             copyFile(file, file.replace(/src([/\\])/, `${options.outputDir}$1`), true, true);
         }
-        // support windows path with "\\" instead of "/"
     }
 };
 
+const currentlyProcessing: string[] = [];

```

Comment:
Could you use a `Set` here?

Reply:
would be an overkill (but the right structure to hold it :-) ). I can change it